monitoring/dashboards/pgmonitor/pg-details.json

```diff
@@ -69,7 +69,7 @@
         "alertThreshold": true
       },
       "percentage": false,
-      "pluginVersion": "8.4.1",
```

Comment:
I wonder if we really needed to touch the dashboard JSON files.  It doesn't look like that was done in previous upgrades to the monitoring components.  I assume it doesn't break anything.  Did you find an issue that updating the dashboards was necessary?

Reply:
I thought that we would need to update the plug-in version with our Grafana version (since the versions numbers matched); however, they were not changed in Bryan's last monitoring update.
I'll revert the **pluginVersion** values in the next push.